Doc/c-api/long.rst

```diff
@@ -571,6 +571,42 @@ distinguished from a number.  Use :c:func:`PyErr_Occurred` to disambiguate.
    .. versionadded:: 3.14
 
 
+.. c:function:: int PyLong_IsPositive(PyObject *obj)
+
+   Check if the integer object *obj* is positive.
+
+   On success, return 1 if *obj* is positive, and 0 otherwise.
+
+   On failure, return -1 with an exception set.  This function always succeeds
+   if *obj* is a :c:type:`PyLongObject` or its subtype.
```

Comment:
I'm not sure if we should include implementation specifics like this. Consider leaving it out. Ditto for the other APIs.

```suggestion
   On failure, return -1 with an exception set.
```

Reply:
I'm considering this, and my response to this question is perhaps affirmative. Since subclasses of Long are still instances of Long, they represent an extension of the Long class while maintaining its behavior. A crucial insight from this explanation is that it is capable of accepting other subclasses of Long, not just the Long class itself. (I think)

Reply:
Oh, no.  Please keep this;)  Removing reduces API convenience a lot, use cases can't be anymore one-liners.  While we not gain anything at all.  Why the hell checking of the sign might fail for integers?